getting_started/dedicated_administrators.adoc

```diff
@@ -0,0 +1,110 @@
+= Administering an {product-title} Cluster
+:data-uri:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As an administrator of a {product-title} cluster, your account has additional
+permissions and access to all user-created projects in your organization's
+cluster. While logged in to an account with this role, the basic
+link:../cli_reference/basic_cli_operations.html[developer CLI] (the `oc`
+command) allows you increased visibility and management capabilities over
+objects across projects, while the
+link:../cli_reference/admin_cli_operations.html[administrator CLI] (commands
+under the `oc adm` command, and formerly the `oadm` command) open up additional
+operations.
+
+[[gs-dedicated-admin-downloading-the-cli]]
+== Downloading the CLI
+
+You can download the CLI used for both normal developer operations and
+administrator operations from the *About* page in the web console. See
+link:../cli_reference/get_started_cli.html[Get Started with the CLI] for more
+detailed installation steps.
+
+[[gs-dedicated-admin-logging-in]]
+== Logging In and Verifying Permissions
+
+To log in as an administrator of a {product-title} cluster, you can log in to
+the web console or CLI just as you would if you were a normal developer user.
+
+When you link:../dev_guide/authentication.html#web-console-authentication[log
+into the web console], you will see all user-created projects across the
+cluster.
```

Comment:
s/login to/log in to

But...I'm not too if the above is terribly relevant. You mention in the Overview para that an admin has access to all projects. Is it needed here?

Also, with the small para below, maybe:

"Use the `oc login` command to log in with the CLI:" or something less wordy.